.github/workflows/build_portable_linux_pytorch_wheels.yml

```diff
@@ -92,9 +109,12 @@ jobs:
           git config --global user.email "therockbot@amd.com"
 
       - name: Select Python version
+        id: select-python
+        shell: bash
         run: |
-          python build_tools/github_actions/python_to_cp_version.py \
-            --python-version ${{ inputs.python_version }}
+          cp_version=$(python build_tools/github_actions/python_to_cp_version.py \
+            --python-version ${{ inputs.python_version }})
+          echo "cp_version=$cp_version" >> "$GITHUB_OUTPUT"
```

Comment:
You don't need to pass this to `cp_version` and it wouldn't be what you expect. E.g.

```bash
cp_version=$(python build_tools/github_actions/python_to_cp_version.py --python-version 3.12)
echo $cp_version
```
gives you
> cp312-cp312 Setting environment variable by appending to $GITHUB_ENV: {'cp_version': 'cp312-cp312'}

However, the script already sets the variable `env.cp_version` which you can use in other (later?) steps.

Reply:
We can also change the script to write to also write to GITHUB_OUTPUT, rather than add a separate step:
```diff
      - name: Select Python version
        id: select-python
        run: |
          python build_tools/github_actions/python_to_cp_version.py \
            --python-version ${{ inputs.python_version }}
-
-      - name: Add cp_version to output
-        id: output-cp-version
-        run:
-          echo "cp_version=${{ env.cp_version }}" >> $GITHUB_OUTPUT
```

https://github.com/ROCm/TheRock/blob/4aba3a7903dd3b50a6ba2d3cb9da78ee0e0bf756/build_tools/github_actions/python_to_cp_version.py#L56-L57 https://github.com/ROCm/TheRock/blob/4aba3a7903dd3b50a6ba2d3cb9da78ee0e0bf756/build_tools/github_actions/github_actions_utils.py#L61-L64